x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/space_management/space_management.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useEffect, useState, FC, useCallback, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiButtonEmpty,
+  EuiSpacer,
+  EuiTabbedContent,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiInMemoryTable,
+  SearchFilterConfig,
+  EuiBasicTableColumn,
+} from '@elastic/eui';
+
+import type { SpacesPluginStart } from '@kbn/spaces-plugin/public';
+import type { JobType, MlSavedObjectType } from '../../../../../../../common/types/saved_objects';
+import type {
+  ManagementListResponse,
+  ManagementItems,
+} from '../../../../../../../common/types/management';
+import { useManagementApiService } from '../../../../../services/ml_api_service/management';
+import { getColumns } from './columns';
+import { MLSavedObjectsSpacesList } from '../../../../../components/ml_saved_objects_spaces_list';
+import { getFilters } from './filters';
+
+interface Props {
+  spacesApi: SpacesPluginStart | undefined;
+  setCurrentTab: (tabId: MlSavedObjectType) => void;
+}
+
+export const SpaceManagement: FC<Props> = ({ spacesApi, setCurrentTab }) => {
+  const { getList } = useManagementApiService();
+  const [currentTabId, setCurrentTabId] = useState<MlSavedObjectType>('anomaly-detector');
+  const [jobs, setJobs] = useState<ManagementListResponse>();
```

Comment:
not sure if `jobs` is an appropriate name, because if I'm not mistaken it could be models as well

Reply:
Changed in https://github.com/elastic/kibana/pull/134893/commits/0c08ba996272059b4ad78b8e9cfd4fbb1b0735bd